examples/config/selinux/go-xdp-counter-sharing-map/kustomization.yaml

```diff
@@ -15,27 +15,27 @@ patches:
       value: {"pod-security.kubernetes.io/enforce":"privileged","pod-security.kubernetes.io/warn":"privileged"}
   target:
     kind: Namespace
-    name: go-xdp-counter
+    name: go-xdp-counter-sm
 - patch: |-
     - op: add
       path: "/spec/template/spec/containers/0/securityContext/seLinuxOptions"
       value: {}
   target:
     kind: DaemonSet
-    name: go-xdp-counter-ds
+    name: go-xdp-counter-sm-ds
 - patch: |-
     - op: add
       path: "/spec/template/spec/containers/0/securityContext/seLinuxOptions/type"
       value:
         bpfman-secure_go-xdp-counter.process
   target:
     kind: DaemonSet
-    name: go-xdp-counter-ds
+    name: go-xdp-counter-sm-ds
 resources:
 - selinux.yaml
 - binding.yaml
 - ../../base/go-xdp-counter-sharing-map
 images:
-- name: quay.io/bpfman-userspace/go-xdp-counter
+- name: quay.io/bpfman-userspace/go-xdp-counter-sharing-map
```

Comment:
For consistency, should this be -sharing-map, or -sm. Having now seen -sharing-map, perhaps explicit is better so s/-sm/-sharing-map/g?